packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -328,7 +395,7 @@ class ExpansionPanelList extends StatefulWidget {
   /// If ExpansionPanelList.radio is used, the callback may be called a
   /// second time if a different panel was previously open. The arguments
   /// passed to the second callback are the index of the panel that will close
-  /// and false, marking that it will be closed.
+  /// and true, marking that it was previously open and is closing.
```

Comment:
I think this is the right thing to do, but this will likely break a lot of people. @goderbauer do you think if this breaking change worth making?